pkg/defaults/defaults.go

```diff
@@ -591,3 +605,27 @@ func defaultImageFromReleaseTag(base api.ImageStreamTagReference, release *api.R
 	base.Namespace = release.Namespace
 	return base
 }
+
+func buildRootImageStreamFromRepository(readFile readFile) (*api.ImageStreamTagReference, error) {
```

Comment:
Why can't we expect structured data here in normal name/namespace/tag format like everywhere else in config?

Reply:
done